### PR TITLE
Add placeholder post file

### PR DESCRIPTION
**Pull request by Robinlovelace**
Heads-up @Nowosad I plan to ask for a review today or tomorrow : )

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Please take a look @Nowosad. I've pencilled in Monday as release date for this, sound reasonable? I reckon a topical example using R-spatial could be worthwhile, just to make the point that R is an outstanding environment for reproducible geographic data analysis code.

**Comment by Robinlovelace** [comment]
Ready for review @Nowosad !

**Comment by Nowosad** [comment]
@Robinlovelace I have read the text, and made some tiny changes. It is good, but I have a strange feeling that there is something missing. (You should not worry about it - I will think about it...). Two small things:

1. I would suggest adding more explanation for readers on what is going on in the code in the first section (1. Installing spatial R packages on Ubuntu)
2. I also think, it would be good to point the readers to places where to look for solutions when the R spatial packages installations fail. (E.g., googling the error message, search in sf issues, etc.)

File: content/post/2020/installing-r-spatial-packages-linux.Rmd [hunk 1]
```diff
@@ -35,30 +35,30 @@ These imminent changes include:
 - Ongoing changes to the OSGeo stack on which key geographic R packages, leading to a huge amount of heroic [r-spatial](https://github.com/r-spatial/) development activity to support the new features, including more precise [coordinate reference system definitions](https://www.r-spatial.org/r/2020/03/17/wkt.html).
 
 This post touches on some of these things but, its main purpose is to **provide advice on getting R's key spatial packages up-and-running on a future-proof Linux operating system** (Ubuntu).^[
-We plan to keep this post updated with new developments such as the release of Ubuntu 20.04 (due to be released [2020-04-23](https://itsfoss.com/ubuntu-20-04-release-features/)) and R 4.0.0 (due to be released the day after, on [2020-04-24](https://developer.r-project.org/)).
```
Comment (Robinlovelace) [inline content/post/2020/installing-r-spatial-packages-linux.Rmd]: :+1:

**Review by Robinlovelace — COMMENTED** [review]
Heads-up @Nowosad I've just responded to your suggestions. I think your changes were great and your suggestions were good. In terms of what's missing... a bit of R code showing a timely issue (e.g. spread of Covid-19)?

**Pull request merged**